### PR TITLE
Feature/trace enable debug

### PR DESCRIPTION
**Pull request by Setogit**
Connect to strongloop-internal/scrum-nodeops#344

### CODE REVIEW

**Comment by Setogit** [comment]
Propose to add U option to strong-pm to enableDebugServer of MinkeLite

**Comment by sam-github** [comment]
@Setogit What is the debug server, and what port does it listen on?

When you base a PR on another, you should mention in the initial comment something like:

Based on https://github.com/strongloop/strong-pm/pull/132, blocked until it is merged.

**Comment by sam-github** [comment]
Lacks usage documentation in bin/sl-pm.txt. I'm not sure what the debug server does, but if its debug-only, I suggest it would be better to enable it via environment. see comment above.

File: lib/server.js [hunk 1]
```diff
@@ -36,8 +36,15 @@ function Server(originalCommand, baseDir, listenPort, controlPath) {
...
 
-  this._serviceManager = new ServiceManager(this);
-  this._app = meshServer(this._serviceManager);
+  this._serviceManager = new ServiceManager(this, !!traceOptions.enableTracing);
+
+  var options = {};
+  if (!!traceOptions.enableTracing) {
+    options['trace.enable'] = true;
+    options['trace.db.path'] = this._baseDir;
+    options['trace.enableDebugServer'] = !!traceOptions.enableDebugServer;
```
Comment (sam-github) [inline lib/server.js]: I suggest the only change necessary for a debug function would be here:
```
options['trace.enableDebugServer'] = !!process.env.STRONGLOOP_DEBUG_MINKELITE;
```

^--- above name is provisional, I'm not sure what the debug server is for.

**Comment by Setogit** [comment]
Based on #132, blocked until it is merged.

**Comment by Setogit** [comment]
connect to strongloop-internal/scrum-nodeops#383

**Comment by kraman** [comment]
Merged into #32

**Pull request closed without merging**